packages/client/src/pages/Onboarding/Onboarding.page.tsx

```diff
@@ -127,6 +129,7 @@ export const OnboardingPage = () => {
           onClick={async () => {
             // TODO: Update in Wizard after submission
             await refetch(); // Update the user preferences
+            LocalStorage.onbordingPreferences = null;
```

Comment:
We should probably only do this on a successful submission. Did you want to update that @tomoyakuroda ?

Reply:
@tristanMatthias  I see. We have updatePreferences mutation in packages\client\src\containers\Wizard.container.ts, so I need to add this code after the mutation, is it correct?

Reply:
Where you have it is fine :). We just want to check if the refetch was _successful_.

EG:
```ts
const res = await refetch();
if (response is a valid req) LocalStorage.onboardingPreferences = null;
else ...
```

Reply:
All right. I just added if statement.